src/flexbox.rs

```diff
@@ -277,7 +277,7 @@ impl Taffy {
                 min_size: child_style.min_size.maybe_resolve(constants.node_inner_size),
                 max_size: child_style.max_size.maybe_resolve(constants.node_inner_size),
 
-                position: child_style.position.zip_size(constants.node_inner_size, |p, s| p.maybe_resolve(s)),
+                position: child_style.position.clone().zip_size(constants.node_inner_size, |p, s| p.maybe_resolve(s)),
```

Comment:
Is it not possible to avoid the clone here? It's being resolved later on this line, which indicates to me that it should be possible. Maybe it's just zip_size method having the wrong signature that's causing the issue here?

Reply:
If we change `zip_size` to take `&self` then we have to clone in the implementation of that function when creating the new `Rect`. Not sure if that's better or worse.

Reply:
```rust
position: Size {
     width: child_style.position.width.maybe_resolve(constants.node_inner_size.width),
     height: child_style.position.height.maybe_resolve(constants.node_inner_size.height),
}
```
ought to work, right? So I think something's being lost in the zip abstraction somewhere.